index.html

```diff
@@ -46,4 +45,241 @@ <h1 class="mola">  I'm Sayed Ramin.<br />Glad to see<br />
+        <h3 class="title1">Tonic</h3>
+        <ul class="flex">
+          <li class="canopy-">CANOPY</li>
+          <li class="p"> <img src="image/works/Counter.png"></li>
+          <li class="role">Back End Dev</li>
+          <li class="p"> <img src="image/works/Counter.png"></li>
+          <li class="year">2015</li>
+        </ul>
+        <p class="text1">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="btn1">
+          <li class="btn">
+            <button class="html" type="button">HTML</button>
+          </li>
+          <li class="btn">
+            <button class="css" type="button">CSS</button>
+          </li>
+          <li class="btn">
+            <button class="java" type="button">JAVASCRIPT</button>
+          </li>
+        </ul>
+        <div class="btn2">
+          <button type="button" class="btn-see">See Project</button>
+        </div>
+      </div>
+    </div>
+    <!--Works Section page 4-->
+    <section class="page4">
+      <div class="@continer">
+    
+        <div class="layout4">
+          <a href="#">
+            <img
+              class="imagework"
+              src="image/works/Snapshoot Portfolio (3).png"
+              alt="background"
+            />
+          </a>
+          <div class="pra">
+            <h3 class="title1">Multi-Post</br> Stories </h3></br></h3>
+            <ul class="flex">
+              <li class="canopy-">CANOPY</li>
+              <li class="p"> <img src="image/works/Counter.png"></li>
+              <li class="role">Back End Dev</li>
+              <li class="p"> <img src="image/works/Counter.png"></li>
+              <li class="year">2015</li>
+            </ul>
+            <p class="text1">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="btn1">
+              <li class="btn">
+                <button class="html" type="button">HTML</button>
+              </li>
+              <li class="btn">
+                <button class="css" type="button">CSS</button>
+              </li>
+              <li class="btn">
+                <button class="java" type="button">JAVASCRIPT</button>
+              </li>
+            </ul>
+            <div class="btn2">
+              <button type="button" class="btn-see">See Project</button>
+            </div>
+          </div>
+        </div>
+ <!--Works Section page 5-->
+ <section class="page5">
+  <div class="@continer">
+      </a>
+      <div class="pra1">
+        <h3 class="titleA">About My self</h3>
+        <p class="text3">
+          Hello I’m a software developer! I can<br> help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h9 class="call1">LET'S CONNECT</h9>
```

Comment:
[OPTIONAL]
- [ ] Nice work making the change suggested by the previous reviewer :clap: However are define with the `h1` to `h6` tags. Kindly use on tags between`h2`,`h3`,`h4`,`h5`,`h6`. You can read more about headings in html [here](https://www.w3schools.com/html/html_headings.asp)